slasher/db/BUILD.bazel

```diff
@@ -0,0 +1,34 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "block_header.go",
+        "db.go",
+        "schema.go",
+    ],
+    importpath = "github.com/prysmaticlabs/prysm/slasher/db",
+    visibility = ["//visibility:public"],
```

Comment:
should the visibility be public here?

Reply:
changed to     visibility = ["//slasher:__subpackages__"],